app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def index
+    # shows index view
+  end
+
+  def user
+    @user_id = params[:id]
+  end
```

Comment:
- [ ] We should not have this method. Kindly rename it to `show`, instead of `user` to match the route defined above.